pkg/services/accesscontrol/resourcepermissions/options.go

```diff
@@ -42,4 +42,6 @@ type Options struct {
 	UidSolver UidSolver
 	// InheritedScopesSolver if configured can generate additional scopes that will be used when fetching permissions for a resource
 	InheritedScopesSolver InheritedScopesSolver
+	// InheritedScopePrefixes if configured are used to create evaluators with the scopes returned by InheritedScopesSolver
+	InheritedScopePrefixes []string
 }
```

Comment:
this is a great set of attributes. Could we explain inheritedScopes a bit more clearer here.

Or at least I am looking for something where we can explain it a bit more in detail and even with a clear example. If someone comes into this permissions system, this gets really hairy quick. ( at least in my opionon ) and these are one of those places.

Maybe you could find a better place to place clear documentation of what inherited permissions mean for children and parents and the distinction between them

Reply:
I agree that it's difficult to wrap your head around. I'm not sure, however, if this is the right place for more detailed documentation.

Once we have a bit more time, we will either try to simplify the access control flow or add proper documentation with diagrams to it. But for now I think we should just go ahead and fix the issues before the release. I know it's creating tech debt for future, but we need to move fairly quickly right now.